### PR TITLE
Flexibus/#374/user dashboard

### LINKED ISSUE #374: User Dashboard
**This depends on #380 for event logic, #379 for team logic and tile component, #378 for card component.**

AS a user
I WANT to see all my events and teams in one place
IN ORDER to be able to have a clear view and manage all my stuff

Dashboard contains information about:

1. Events that user created or joined (as a player or as a team, where he is a member).
2. Teams that user created or joined.
3. The newest events that were created on platform.
4. (Static) block "Create event".

- reuse card component from #378 
- reuse tile component from #379

Tasks:
- [x] create team tiles
- [ ] create event tiles
- [ ] "create event" button

In wide-screens every block has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row.
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE  - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

Link "Alle anzeigen" goes to "/events" or "/teams" without any modification of these pages or personalisation. Just static link.

The whole block (pic + name + description) is a link and it goes to corresponding event or team.

Header and Footer are here out of scope!

If user has no events or/and no teams, then "Deine Events" or/and "Deine Teams" will be not shown at all.
"Neuste Events" and "Event create" will be shown always.

It looks like this:

### PR DESCRIPTION
**Pull request by emanuel-metzenthin**
Closes #374.

The dashboard is not finished. But it already contains the user's teams. This can be merged before the feature freeze and final presentation.

### CODE REVIEW

File: app/views/devise/registrations/dashboard.html.erb [hunk 1]
```diff
@@ -2,3 +2,14 @@
 <div class="page-header">
   <h1><%= t('dashboard.title', name: @user.first_name) %></h1>
 </div>
+
+<% if user_signed_in? and current_user.non_single_teams.present? %>
```
Comment (jak-ing) [inline app/views/devise/registrations/dashboard.html.erb]: Why do you check for `user_signed_in` although the user is redirected to a 40x page anyways if not signed in and trying to access the dashboard?

File: config/locales/de.yml [hunk 2]
```diff
@@ -111,7 +111,8 @@ de:
       edit_profile: "Profil bearbeiten"
 
   dashboard:
-    title: "Dashboard für %{name}"
+    title: "Hallo %{name}!"
+    show_all: "Alle anzeigem"
```
Comment (jkbe) [inline config/locales/de.yml]: Typo

**Review by jkbe — CHANGES_REQUESTED** [review]
Typo

File: app/views/devise/registrations/dashboard.html.erb [hunk 3]
```diff
@@ -2,3 +2,14 @@
 <div class="page-header">
   <h1><%= t('dashboard.title', name: @user.first_name) %></h1>
 </div>
+
+<% if user_signed_in? and current_user.teams_created_by_user.present? %>
+  <h4><%= t('teams.index.my_teams')%> <a><%= t('dashboard.show_all')%></a></h4>
```
Comment (jkbe) [inline app/views/devise/registrations/dashboard.html.erb]: `href` is missing -> link doesn't work

**Review by jkbe — CHANGES_REQUESTED** [review]

**Comment by jkbe** [comment]
Number of teams shown is not limited to 4 - is that inteded or ok for now? @emanuel-metzenthin @zhamanakov 
image

**Comment by emanuel-metzenthin** [comment]
Number of teams is ok, as no other sections are implemented yet.

**Review by jkbe — APPROVED** [review]

**Pull request merged**